src/palindrom.js

```diff
@@ -487,12 +487,20 @@ var Palindrom = (function() {
     reconnectionFn(function bootstrap(json) {
       palindrom.reconnector.stopReconnecting();
 
-      palindrom.queue.reset(palindrom.obj, json);
-
       if (palindrom.debug) {
         palindrom.remoteObj = JSON.parse(JSON.stringify(json));
       }
 
+      if(!palindrom.isObjectProxified) { // first time the state is being set
```

Comment:
Do we really need to observe to ubobserve just after? Looks like a not needed but expensive operation.